inffast.c

```diff
@@ -259,18 +259,18 @@ void Z_INTERNAL zng_inflate_fast(PREFIX3(stream) *strm, unsigned long start) {
                     }
                     if (op < len) {             /* still need some from output */
                         len -= op;
-                        out = functable.chunkcopy_safe(out, from, op, safe);
+                        out = functable.chunkcopy_safe(out, from, op, (uint32_t)(safe - out) + 1);
                         out = functable.chunkunroll(out, &dist, &len);
-                        out = functable.chunkcopy_safe(out, out - dist, len, safe);
+                        out = functable.chunkcopy_safe(out, out - dist, len, (uint32_t)(safe - out) + 1);
                     } else {
-                        out = functable.chunkcopy_safe(out, from, len, safe);
+                        out = functable.chunkcopy_safe(out, from, len, (uint32_t)(safe - out) + 1);
```

Comment:
Missing extra `()` in cast... Should use widest possible type as long as possible.

Reply:
Fixed thanks.